packages/bits/demo/src/components/demo/wizard-v2/wizard-custom-layout/wizard-custom.component.less

```diff
@@ -10,11 +10,16 @@
     .nui-wizard-step-header {
         border: @nui-line-default solid;
         border-radius: 5px;
-
         .setCssVariable(border-color, nui-color-line-default);
         .setCssVariable(background, nui-color-bg-content);
-
+        flex-direction: row-reverse;
+        justify-content: flex-end;
         padding: 0 8px;
+
+        .nui-icon{
+            transform: translate(-70%, -55%);
```

Comment:
The consumer really shouldn't have to reposition an icon like this. I think a further investigation on the root cause of the misalignment would be better. It may have something to do with the fact that the circle used for the icon is 20px instead of the typical icon size of 16px.